homeworks/05-redux-basic/redux-basic/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
duplicate folder structure:
`05-redux-basic`
`redux-basic`

Please move all files to `05-redux-basic` and remove sub-folder `redux-basic`